spec/models/contributor_spec.rb

```diff
@@ -77,10 +77,49 @@
     end
   end
 
+  describe '#signal_phone_number' do
+    it 'strips whitespaces' do
+      contributor = create(:contributor, signal_phone_number: ' +49 123 456 789  ')
+      expect(contributor.signal_phone_number).to eq '+49123456789'
+    end
+
+    it 'strips slashes, hypens and parantheses' do
+      contributor = create(:contributor, signal_phone_number: '+49(0)/123-456789')
+      expect(contributor.signal_phone_number).to eq '+49123456789'
+    end
+
+    it 'replaces leading zeros' do
+      contributor = create(:contributor, signal_phone_number: '0049123456789')
+      expect(contributor.signal_phone_number).to eq '+49123456789'
+    end
+
+    it 'assumes German country code by default' do
```

Comment:
maybe add a test for accepting other country codes / storing numbers with the given country code if one is given

Reply:
This is done a few lines below, no?